packages/amplication-client/src/Plugins/InstalledPluginSettings.tsx

```diff
@@ -186,7 +186,9 @@ const InstalledPluginSettings: React.FC<Props> = ({
           </div>
           <HorizontalRule />
           <CodeEditor
-            defaultValue={pluginInstallation?.PluginInstallation.settings}
+            defaultValue={JsonFormatting(
+              pluginInstallation?.PluginInstallation.settings
+            )}
             resetKey={resetKey}
             onChange={onEditorChange}
```

Comment:
Let's format on change too.

Reply:
sure

Reply:
@overbit i have commit change can you review now